src/vuestic-theme/vuestic-components/vuestic-color-picker/VuesticSimpleColorPicker.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="vuestic-simple-color-picker">
+    <ul class="vuestic-colors" style="padding-top: 15px">
+      <span class="vuestic-color-item dot" v-for="c in palette"
+            :aria-label="'color:' + c"
+            @click="handlerClick(c)"
+            :class="{'dot-selected': isSelected(c)}"
+            :style="{background: c}">
+      </span>
+    </ul>
+  </div>
+</template>
+
+<script scoped>
+
+  const defaultColors = [
```

Comment:
Why would you need that?

Reply:
to provide default colors for this component.

Reply:
Awesome. =.=

Second question - Why would you need default colors for this component? Do you expect user to use colors you provided? What for?